apps/settings/js/modules/bluetooth/bluetooth_context.js

```diff
@@ -749,6 +771,35 @@ define(function(require) {
     },
 
     /**
+     * Given paired, address properties to find out device element 
+     * from remote/paired devices list.
+     *
+     * @access private
+     * @memberOf BluetoothContext
+     * @param {String} address
+
+     * @param {Object} options
+     * @param {String} options.paired - is paired or not
+     * @param {String} options.address - the address of the device
+     * @return {Object} device
+     */
+    _findDeviceByAddress:
```

Comment:
I think it is okay not to take `paired` as a parameter. Address is sufficient for finding the device as there are only two device lists to be searched. This also makes the function easier to use.